### DISCUSSION TITLE
Question about Coverage Map for UAV at a fixed altitude

### DISCUSSION THREAD

**Discussion by Liu-Zhilin** [NVlabs/sionna#715] [Q&A] [answered]
As a newcomer to Sionna and coverage map generation, I'm interested in creating a coverage map for UAV navigation at a fixed altitude of 100 meters. How can I achieve this?

**Comment by merlinND** [comment, marked as answer]
Hello @Liu-Zhilin,

Have you tried using the `cm_center` argument of the `Scene.coverage_map()` function to center the coverage map at your desired altitute?
https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Scene.coverage_map

**Comment by Liu-Zhilin** [comment]
Thank you very much! I have resolved this problem.





---- Replied Message ----
| From | Merlin ***@***.***> |
| Date | 01/03/2025 18:15 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [NVlabs/sionna] Question about Coverage Map for UAV at a fixed altitude (Discussion #715) |

Hello @Liu-Zhilin,

Have you tried using the cm_center argument of the Scene.coverage_map() function to center the coverage map at your desired altitute?
https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Scene.coverage_map

**Reply by Advaith2212** [reply]
Can you kindly let me know the setup of UAV you are considering for modelling. 

Thanks and regards,
Advaith
